index.html

```diff
@@ -31,5 +31,11 @@ <h4 class="text feature-title"> No internet needed. </h4>
         </div>
       </div>
     </div>
+    <div class="footer">
+      <h4 class="subscription-title font white-font"> Get info on new updates </h4>
+      <input class="subscription-input rounded-input" type="text">
```

Comment:
since this is the subscription form the input and the button should be wrapped inside a form tag